### ISSUE TITLE
Make an issue template for mentorship requests

### ISSUE THREAD

**Issue by mbestavros** [enarx/enarx#418]
With some newcomers showing interest in the project (hooray!), we'd like to expose a mechanism to provide mentorship if wanted. Creating a new issue with a mentorship request would work well with our existing issue triage framework; we should additionally provide a template to make filling out such an issue easy.

**Comment by MikeCamel** [comment]
Let's get this moving.  Don't want to stall hopeful people (or we could just assign mentors in parallel).
